cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -123,6 +126,21 @@ func (p *Provisioner) ProvisionHostPath(opts pvController.VolumeOptions, volumeC
 	return pvObj, nil
 }
 
+// GetNodeObjectFromHostName returns the Node Object with matching NodeHostName
+func (p *Provisioner) GetNodeObjectFromHostName(hostName string) (*v1.Node, error) {
+	labelSelector := metav1.LabelSelector{MatchLabels: map[string]string{persistentvolume.KeyNode: hostName}}
+	listOptions := metav1.ListOptions{
+		LabelSelector: labels.Set(labelSelector.MatchLabels).String(),
+		Limit:         1,
```

Comment:
Do we need the `Limit` here?

Reply:
There should be only one node, right? Rather than explicitly writing an error handling, good to limit the number of nodes right here. Right?